tardis/model/base.py

```diff
@@ -23,6 +23,11 @@
 logger = logging.getLogger(__name__)
 
 
+class ModelState():
+    def __init__(self):
```

Comment:
you want to initialize it with the variables as discussed

Reply:
Is https://github.com/tardis-sn/tardis/pull/1847/commits/2e58f013a97203ed867c823bf7348b7a8d4f0e5e a good way to do this? Because properties are very related to each other, I thought it might be good to move those calculations to `model_state` class.